content/en/docs/instrumentation/python/mypy.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Using mypy
+weight: 5
+---
+
+If you're using [mypy](http://mypy-lang.org/), you'll need to turn on [namespace
+packages](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-namespace-packages).
+
+In your project configuration file, add the following:
+
+```toml
+[tool.mypy]
+namespace_packages = true
+```
+
+Or if you'd prefer a command-line switch, use `--namespace-packages`:
+
+```console
```

Comment:
Since you're not adding a leading prompt, you can make the block language `shell` (or add `$` as a prompt).